src/openrct2/scripting/bindings/game/ScContext.hpp

```diff
@@ -45,6 +45,36 @@ namespace OpenRCT2::Scripting
         }
 
     private:
+        std::vector<DukValue> allPlugins_get()
+        {
+            // Get all of the plugins from the script engine
+            OpenRCT2::Scripting::ScriptEngine& scriptEngine = GetContext()->GetScriptEngine();
+            std::vector<std::shared_ptr<OpenRCT2::Scripting::Plugin>> allPlugins = scriptEngine.GetPlugins();
+            auto ctx = scriptEngine.GetContext();
+            std::vector<DukValue> result;
+            // Iterate through all plugins and and cast their data to Duk objects
+            for (std::shared_ptr<OpenRCT2::Scripting::Plugin> plugin_ptr : allPlugins)
```

Comment:
```suggestion
            for (const std::shared_ptr<OpenRCT2::Scripting::Plugin>& plugin_ptr : allPlugins)
```

It'll probably make the line below fail to compile, but adding constness to it too might help.